### PR TITLE
executor: remove Next() function for TableScanExec

### PR DESCRIPTION
**Pull request by qxhy123**

### CODE REVIEW

File: executor/executor.go [hunk 1]
```diff
@@ -905,50 +905,12 @@ type TableScanExec struct {
 	columns    []*model.ColumnInfo
 
 	isVirtualTable     bool
-	virtualTableRows   [][]types.Datum
-	virtualTableCursor int
 
 	// for chunk execution
```
Comment (XuHuaiyu) [inline executor/executor.go]: This comment can be removed.

**Comment by zz-jason** [comment]
@qxhy123 you can run `make check` to fix ci.

**Comment by qxhy123** [comment]
@zz-jason thanks

**Comment by XuHuaiyu** [comment]
LGTM

**Pull request merged**